bench/irmin-pack/trace_replay.ml

```diff
@@ -416,12 +402,16 @@ module Make (Store : Store) = struct
                  has idx %d with key %a"
                 gc_commit_idx pp_key gc_commit_key (i - 1) pp_key
                   (Hashtbl.find t.key_per_commit_idx (i - 1))];
-              Store.gc repo gc_commit_key)
+              let finished = function
+                | Ok elapsed ->
+                    [%logs.app
+                      "Gc ended on commit idx %d with key %a, it took %.4fms"
+                        gc_commit_idx pp_key gc_commit_key elapsed]
```

Comment:
This commit info is for the current commit, that is the commit that calls the gc and not the one which ends it, no?

Reply:
Ah, good catch. I thought the gc end log was to correspond with the log for gc start but now see it logs the commit the finalization happens. Is it important to restore this functionality? There is no knowledge of current `i` in this function when it is called, but I could (perhaps) add a global ref to track current commit idx. Otherwise, I could update this log to say "Gc ended for" instead of "on".

Reply:
It was a nice info to have. Without it, while running the benchmarks, we only know how much time a gc took, but not how many block were validated during the gc, or how spaced out (in terms of blocks) the gcs are. But if its too cumbersome, you can also just switch  to "Gc ended for".

Reply:
Thanks, that's helpful context. I'll take a look at restoring this.

Reply:
Restored what I am calling "commit duration" to the gc ended log